gtav/pillbox_hospital.lua

```diff
@@ -4,9 +4,31 @@ exports('GetPillboxHospitalObject', function()
 end)
 
 PillboxHospital = {
-    ipl = "rc12b_default",
+    iplFixed = 'RC12B_Fixed',
+    iplDefault = 'RC12B_Default',
+    iplDestroyed = 'RC12B_Destroyed',
+    iplHospitalInterior = 'RC12B_HospitalInterior',
 
-    Enable = function(state)
-        EnableIpl(PillboxHospital.ipl, state)
+    LoadDefault = function(state)
+        PillboxHospital.DisableIpl()
+        EnableIpl(PillboxHospital.iplDefault, state)
+    end,
```

Comment:
Either way, it will be a breaking change I believe. If you do a LoadDefault then the broken entityset can be loaded together instead of 2 enables. However, if you stick with the enable then you must load the entitysets by calling them rather than just doing it through a boolean. Which is the best course of action?